packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -18,19 +19,19 @@ export interface TabProps extends HTMLProps<HTMLAnchorElement> {
   onChangeTab?: (event?: React.MouseEvent<HTMLAnchorElement>) => void;
   /** A number rendered next to the text. Usually used to display the number of items in a tab's view. */
   counter?: number | null;
-  suffix?: () => JSX.Element;
+  suffixText?: string;
```

Comment:
Actually I'm not sure about this change because it will make it harder to emit feature highlights events and the Pro badge is a very specific use case but I could see other information we'd like to display as a suffix (like highlighting a new feature with only a blue dot instead of a green badge for example). WDYT?

Reply:
Good point about adding extra info, but I think we could leave this for a second pass over this feature and at first have a more straightforward implementation. 

About emitting the events, I don't think having custom badge will be of much help here since the users click on the actual tab and not the badge. To emit custom events I think we'd need to extend `onChangeTab` of the Tab component.